server/services/prisonService.ts

```diff
@@ -157,7 +157,7 @@ export default class PrisonService {
     return prisonName
   }
 
-  private async getPrisonNames(username: string): Promise<Record<string, string>> {
+  async getPrisonNames(username: string): Promise<Record<string, string>> {
```

Comment:
Exposing previously internal cache of `prisonId: prisonName` key/value pairs as it's easy to pass to template to lookup and add prison names.